packs/elastic/scripts/logsfeed.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
I'm slightly confused - what changes if we move it to etc/ directory?

In any case, not moving that file is not a blocker right now so if it makes things easier, do leave it there.

On a related note - similar as other scripts, docker-compose.yaml is also not really useful without docs. It would be great to add some docs on how to use it to the readme.